### DISCUSSION TITLE
Ganache using some vulnerable library

### DISCUSSION THREAD

**Discussion by sagarkhatri200** [ConsenSys-archive/ganache#3927] [General]
I feel ganache has some security vulnerabilities.  Any data that ganache uses locally should not be exploited or exported over.

**Comment by davidmurdoch** [comment]
Can you prove it?

**Comment by sagarkhatri200** [comment]
Let me do that. I am trying to trace back the transactions. I think i have located the real account linked with the destination account.

In the meantime, do not use real secret on ganache.
This was the only place i used my real secret.

Sagar Khatri


On Nov 18, 2022, at 8:56 AM, David Murdoch ***@***.***> wrote:

﻿

Can you prove it?

**Reply by davidmurdoch** [reply]
Please email [SecOps@consensys.net](mailto:secops@consensys.net) if you discover an issue. Please disclose responsibly.

**Comment by sagarkhatri200** [comment]
+ ***@***.******@***.***>

is the ganache application code being reviewed security wise.i.e code scan, manual vertification of all third party dependencies and probably outbound traffic scan.

If this is happening, then it should be good enough.
if there is an issue, it needs to be addressed.

Sagar Khatri


On Nov 18, 2022, at 10:05 AM, David Murdoch ***@***.***> wrote:

l ***@***.******@***.***>

**Reply by davidmurdoch** [reply]
It is scanned regularly and dependencies are manually verified. We don't monitor outbound traffic.

You do know Ganache isn't a secure wallet or password manager, right?
